### DISCUSSION TITLE
MacOS agent to be listed in Applications

### DISCUSSION THREAD

**Discussion by y0d4a** [wazuh/wazuh#20130] [Ideas]
Hi, MDM solutions do not see Wazuh that is installed because he is not in the list of Applications (do not have packageid), is it possible to add it there?

Thank you

**Comment by MiguelazoDS** [comment]
Hi @y0d4a, 

There's a development around that issue https://github.com/wazuh/wazuh/issues/15798, but there are other implementations with higher priorities so this fix is on hold at this moment. Also, there is an impact on the performance due to recursion in several paths that need revision. 
Apologize for the inconvenience.

**Comment by y0d4a** [comment]
oh, great, did not know that is in plan, great!
thank you for the answer, wish you all the best.

**Reply by MiguelazoDS** [reply]
Thanks to you for your patience. Have a good day!
